### PR TITLE
R4 [ChargeItem] : Adding search by account for chargeItem

### PR DESCRIPTION
**Pull request by babukarthiks**
Description
----
Added information about account parameter in the chargeItem search.

'reference' refers to search parameter
"Reference" refers to the expected object in response

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
Before changes merged screenshot

- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/financial/general/charge-item.md [hunk 1]
```diff
@@ -83,19 +83,22 @@ Search for ChargeItems that meet supplied query parameters:
...
- Name                  | Required?       | Type          | Description
------------------------|-----------------|---------------|-------------------------------------------------------
- `_id`                 | This or context | [`token`]     | The logical resource id associated with the ChargeItem.
- `context`             | This or _id     | [`token`]     | Encounter associated with event.
- `-status`             | See notes       | [`token`]     | The status of the ChargeItem. Example: `billable`
- [`_count`]            | No              | [`number`]    | The maximum number of results to return.
+ Name                  | Required?                  | Type          | Description
+-----------------------|----------------------------|---------------|-------------------------------------------------------
+ `_id`                 | This or context or account | [`token`]     | The logical resource id associated with the ChargeItem.
+ `context`             | This or _id or account     | [`reference`] | Encounter associated with event.
```
Comment (alexbeckner1) [inline content/millennium/r4/financial/general/charge-item.md:89]: Not related to this change, but can we go ahead and change this Description to "Encounter associated with the ChargeItem."

File: content/millennium/r4/financial/general/charge-item.md [hunk 2]
```diff
@@ -61,11 +61,11 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/St
...
  `net-price`            | [`Money`]                         | The quantity times the unit price for a resource (total price).
- `offset-by`            | [`Reference`]                     | Indicates a resource that this resource is offset by. This resource is no longer active when offset.
- `performing-location`  | [`Reference`]                     | A location where the resource was performed.
+ `offset-by`            | [Reference]                       | Indicates a resource that this resource is offset by. This resource is no longer active when offset.
+ `performing-location`  | [Reference]                       | A location where the resource was performed.
  `priority`             | [`unsignedInt`]                   | The priority of the element within a list.
  `procedure`            | None (contains nested extensions) | Procedure performed on the patient associated to the resource.
- `replacing`            | [`Reference`]                     | A reference to a resource that this resource is replacing.
+ `replacing`            | [Reference]                       | A reference to a resource that this resource is replacing.
```
Comment (exopsRamba) [inline content/millennium/r4/financial/general/charge-item.md:68]: What was the reason for changing this? It is [`Reference`] for other resources.

File: content/millennium/r4/financial/general/charge-item.md [hunk 3]
```diff
@@ -83,19 +83,22 @@ Search for ChargeItems that meet supplied query parameters:
...
------------------------|-----------------|---------------|-------------------------------------------------------
- `_id`                 | This or context | [`token`]     | The logical resource id associated with the ChargeItem.
- `context`             | This or _id     | [`token`]     | Encounter associated with event.
- `-status`             | See notes       | [`token`]     | The status of the ChargeItem. Example: `billable`
- [`_count`]            | No              | [`number`]    | The maximum number of results to return.
+ Name                  | Required?                  | Type          | Description
+-----------------------|----------------------------|---------------|-------------------------------------------------------
+ `_id`                 | This or context or account | [`token`]     | The logical resource id associated with the ChargeItem.
+ `context`             | This or _id or account     | [`reference`] | Encounter associated with event.
+ `account`             | This or context or _id     | [`reference`] | Account associated with the ChargeItem.
```
Comment (exopsRamba) [inline content/millennium/r4/financial/general/charge-item.md:90]: nit: let's have parameters in the same order as context. i.e.
```ruby
This or _id or context
```

File: content/millennium/r4/financial/general/charge-item.md [hunk 4]
```diff
@@ -83,19 +83,22 @@ Search for ChargeItems that meet supplied query parameters:
...
+ `-status`             | See notes                  | [`token`]     | The status of the ChargeItem. Example: `billable`
+ [`_count`]            | No                         | [`number`]    | The maximum number of results to return.
 
 Notes:
 
 - When searching by `_id`, only a single ChargeItem id can be provided.
 - `-status` must be provided with `context`.
+- `-status` must be provided with `account`.
```
Comment (exopsRamba) [inline content/millennium/r4/financial/general/charge-item.md:98]: Let's combine these. i.e.
```
`-status` must be provided with `context` and `account`.
```

File: content/millennium/r4/financial/general/charge-item.md (hunk 2 above)
Comment (babukarthiks) [reply content/millennium/r4/financial/general/charge-item.md:68]: We needed multiple reference links(as one is Object and the other is just search type) 
I referring the object with [Reference] and type as [`reference`]. I couldn't have both as the type as the parser is only not case-sensitive and is treating `Reference` and `reference` the same.

File: content/millennium/r4/financial/general/charge-item.md (hunk 3 above)
Comment (babukarthiks) [reply content/millennium/r4/financial/general/charge-item.md:90]: Addressed in 2e4df3f24bdeb9538a6e6875435ea5f695bd59b5

File: content/millennium/r4/financial/general/charge-item.md (hunk 4 above)
Comment (babukarthiks) [reply content/millennium/r4/financial/general/charge-item.md:98]: Addressed in 2e4df3f24bdeb9538a6e6875435ea5f695bd59b5

File: content/millennium/r4/financial/general/charge-item.md (hunk 1 above)
Comment (babukarthiks) [reply content/millennium/r4/financial/general/charge-item.md]: Addressed in 2e4df3f24bdeb9538a6e6875435ea5f695bd59b5

File: content/millennium/r4/financial/general/charge-item.md (hunk 3 above)
Comment (exopsRamba) [reply content/millennium/r4/financial/general/charge-item.md:90]: That commit doesn't address this.

File: content/millennium/r4/financial/general/charge-item.md (hunk 4 above)
Comment (exopsRamba) [reply content/millennium/r4/financial/general/charge-item.md:98]: This commit doesn't have this change.

File: content/millennium/r4/financial/general/charge-item.md (hunk 3 above)
Comment (babukarthiks) [reply content/millennium/r4/financial/general/charge-item.md]: 24bcb410398f2d17801fb6aec8534be947458102

File: content/millennium/r4/financial/general/charge-item.md (hunk 4 above)
Comment (babukarthiks) [reply content/millennium/r4/financial/general/charge-item.md]: 24bcb410398f2d17801fb6aec8534be947458102

File: content/millennium/r4/financial/general/charge-item.md (hunk 2 above)
Comment (babukarthiks) [reply content/millennium/r4/financial/general/charge-item.md:68]: there is only one Reference to that link. think we have multiple references to that object extracting it as a variable looks better.

Comment (Coffeevee) [reply content/millennium/r4/financial/general/charge-item.md:68]: I think for the capitalized `[Reference]` fields, let's try to implement the hotlinks the way Keifer suggested here: https://github.com/cerner/fhir.cerner.com/blame/main/content/millennium/r4/base/management/encounter.md#L68

We can leave the lowercase hotlink implementation untouched similar to what the Encounter resource is doing already.

File: content/millennium/r4/financial/general/charge-item.md (hunk 1 above)
Comment (alexbeckner1) [reply content/millennium/r4/financial/general/charge-item.md]: Thanks for making that change.

File: content/millennium/r4/financial/general/charge-item.md (hunk 2 above)
Comment (babukarthiks) [reply content/millennium/r4/financial/general/charge-item.md]: addressed: 4f6b0836109ab36d830dac57bcc5b24712ff147b

**Review by Anurag-Balannagari — APPROVED** [review]

File: content/millennium/r4/financial/general/charge-item.md [hunk 5]
```diff
@@ -55,19 +55,19 @@ The following fields are returned if valued:
 All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/StructureDefinition/{id}`
 
  ID                     | Value\[x] Type                    | Description
```
Comment (exopsRamba) [inline content/millennium/r4/financial/general/charge-item.md:57]: These column titles look like they need there width changed to align with the changes made below.

**Comment by exopsRamba** [comment]
Let's update the screen shots before final approval.

File: content/millennium/r4/financial/general/charge-item.md (hunk 5 above)
Comment (babukarthiks) [reply content/millennium/r4/financial/general/charge-item.md]: 99c460943bac20003ddaf8ed1d5e505d211da8e7

**Comment by babukarthiks** [comment]
> Let's update the screen shots before final approval.

Updated the screenshot of new changes.

**Review by exopsRamba — APPROVED** [review]

**Review by alexbeckner1 — APPROVED** [review]

**Review by Coffeevee — APPROVED** [review]

**Review by mayur-rajendran — APPROVED** [review]

**Pull request merged**